contracts/modules/STO/USDTieredSTOFactory.sol

```diff
@@ -32,17 +34,19 @@ contract USDTieredSTOFactory is ModuleFactory {
         compatibleSTVersionRange["upperBound"] = VersionUtils.pack(uint8(0), uint8(0), uint8(0));
     }
 
-     /**
+    /**
      * @notice Used to launch the Module with the help of factory
      * @return address Contract address of the Module
      */
-    function deploy(bytes _data) external returns(address) {
+    function deploy(bytes calldata _data) external returns(address) {
         address polyToken = _takeFee();
-        address usdTieredSTO = new USDTieredSTOProxy(msg.sender, polyToken, logicContract);
+        address usdTieredSTO = address(new USDTieredSTOProxy(msg.sender, polyToken, logicContract));
         //Checks that _data is valid (not calling anything it shouldn't)
         require(Util.getSig(_data) == IBoot(usdTieredSTO).getInitFunction(), "Invalid data");
+        bool success;
         /*solium-disable-next-line security/no-low-level-calls*/
-        require(address(usdTieredSTO).call(_data), "Unsuccessfull call");
+        (success, ) = address(usdTieredSTO).call(_data);
```

Comment:
Don't think we need the address cast here.

Reply:
fixed.